src/i18n/translations/messages_hi.json

```diff
@@ -130,5 +130,5 @@
   "Active": "सक्रिय उपयोगकर्ता",
   "Idle": "दूर उपयोगकर्ताओं",
   "Offline": "ऑफ़लाइन उपयोगकर्ता",
-  "Do not mark messages read on scroll": "Do not mark messages read on scroll"
+  "Do not mark messages read on scroll": "स्क्रॉल पर पढ़े संदेशों को चिह्नित नहीं करें"
```

Comment:
Don't you think using निशान instead of चिह्नित would be better?

Reply:
No. I think चिह्नित is better.